src/main/java/org/jenkinsci/plugins/docker/workflow/client/DockerClient.java

```diff
@@ -136,7 +136,7 @@ public String run(@Nonnull EnvVars launchEnv, @Nonnull String image, @CheckForNu
      * @param containerId The container ID.
      */
     public void stop(@Nonnull EnvVars launchEnv, @Nonnull String containerId) throws IOException, InterruptedException {
-        LaunchResult result = launch(launchEnv, false, "stop", containerId);
+        LaunchResult result = launch(launchEnv, false, "stop", "--time=1", containerId);
```

Comment:
Yes…for now, until I can dig into the root problem, I would rather waste the second in the interest of _offering_ a clean shutdown.